repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-intranet-folder.yaml

```diff
@@ -0,0 +1,32 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:queries/hippo:templates/new-intranet-folder:
+      /hippostd:templates:
+        /hippostd:folder:
+          hippostd:foldertype:
+          - new-intranet-folder
+          - new-intranet-document
+          - new-task-folder
```

Comment:
Do we need special queries for `new-task-folder` and `new-internal-news-folder` ? I thought those two doctypes can be created wherever, so no need for special queries for the folders

Reply:
thats probably correct, I'll get rid of them